docs/tutorials/command_line_client.md

```diff
@@ -42,7 +42,7 @@ synapse [-h] [--version] [-u SYNAPSEUSER] [-p SYNAPSEPASSWORD] [-c CONFIGPATH] [
 |---------------------|---------|---------------------------------------------------------------------------|-----------------------|
 | `--version`         | Flag    | Show program’s version number and exit                                    |                       |
 | `-u, --username`    | Option  | Username used to connect to Synapse                                       |                       |
```

Comment:
I was thinking about this. We currently use it if it's passed in to check against the bearer token that the usernames match. It's only used as an optional guard. If I were to remove it I would remove username from everything (Client.py, config file, cli)

Thoughts on removing it entirely?

Reply:
One "issue" is that we didn't explicitly state that we were removing "usernames" in this version. I'm just thinking of users that do this in their code:

```
syn.login(username, authToken =" PAT")

synapse -u username -p PAT
```

On top of that, let's leave it for now in case we wanted to add "oauth" flow that would allow a re-direct to synapse.org to login.

Reply:
Created https://sagebionetworks.jira.com/browse/SYNPY-1393